lib/feature.js

```diff
@@ -0,0 +1,201 @@
+const _ = require('underscore');
+const Backbone = require('backbone');
+
+let API_URL = '';
+let Sale = '';
+let Artworks = '';
+let FeaturedSet = '';
+let FeaturedLinks = '';
+
+module.exports = function (a, b, c, d, e) {
+  API_URL = a;
+  Sale = b;
+  Artworks = c;
+  FeaturedSet = d;
+  FeaturedLinks = e;
+  return module.exports.methods;
+};
+
+//
+// Mixins for the feature page so we can at least keep them consistent across microgravity and force
+module.exports.methods = {
+
+  // Goes down the rabbit hole of APIs necessary to retrieve the data needed to render a
+  // feature. Returns an Array of hashes describing these items such as a collection
+  // of artworks from a sale, or a collection of featured links. And example would look like:
+  //
+  // [
+  //   {
+  //     type: 'sale artworks',
+  //     data: Backbone Collection of artworks
+  //   },
+  //   {
+  //     type: 'featured links',
+  //     data: Backbone Collection of featured links
+  //   }
+  // ]
+  //
+  // Currently the types are:
+  //   * "sale artworks"    Artworks collection
+  //   * "featured links"   FeaturedLinks collection
+  //   * "auction artworks" Artworks collection from an auction
+  //   * "videos"           Like a featured link with videos instead of an images
+  //
+  // @param {Object} options Provide `success` and `error` callbacks similar to Backbone's fetch
+  // @return {Array} An array `Set`s
+
+  fetchSets(options) {
+    if (options == null) {
+      options = {};
+    }
+    return this.fetchSetsAndItems({
+      success: setItems => {
+        // Now that we know how many items need to be mapped, create a callback for those items
+        // that need even further data fetched such as a sale's artworks.
+        const allItemsLen = _.flatten(_.pluck(setItems, 'item')).length;
+        let err = null;
+        const callback = _.after(allItemsLen, () => {
+          if (err) {
+            return options.error(err);
+          }
+          return (typeof options.success === 'function' ? options.success(sets || this.setsFromSetItems(setItems)) : undefined);
+        });
+
+        for (let {
+            orderedSet,
+            items
+          } of Array.from(setItems)) {
+          var sale;
```

Comment:
Curiosity: why are you using `var` here instead of `let`? Or is this some decaff tool deciding to use var?

Reply:
It's a side effect of running `decaffeinate`, I'm not really sure how it chooses `let` vs `var`. Overall it's no different than how CoffeeScript would have compiled it previously.